File: Core/Entities/LoanApplication.cs
namespace Core.Entities
{
    public class LoanApplication
    {
        public int ApplicationID { get; set; }
        public int CustomerID { get; set; }
        public int LoanProductID { get; set; }
        public decimal RequestedAmount { get; set; }
        public DateTime ApplicationDate { get; set; }
        public string ApplicationStatus { get; set; }
        public int? AssignedEmployeeID { get; set; }
        public DateTime? DecisionDate { get; set; }
        public decimal? ApprovedAmount { get; set; }
        public string RejectionReason { get; set; }
    }
}

File: Angular/loan-management-system/src/app/models/loan-application.ts
export interface LoanApplication {
  applicationId: number;
  customerId: number;
  loanProductId: number;
  requestedAmount: number;
  applicationDate: string;
  applicationStatus: string;
  assignedEmployeeId: number | null;
  decisionDate: string | null;
  approvedAmount: number | null;
  rejectionReason: string | null;
}
